components/org.wso2.carbon.business.rules.core/pom.xml

```diff
@@ -179,7 +179,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-checkstyle-plugin</artifactId>
-                <configuration><skip>false</skip></configuration>
+                <!--<configuration><skip>false</skip></configuration>-->
```

Comment:
Get rid of this, if not being used.

Reply:
This was removed by ddc2b176a678135cafd00324e2b24fd2d94fe436